### ISSUE TITLE
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed 

### ISSUE THREAD

**Issue by Michael2008S** [paritytech/parity#7835]
I'm running:

 - **Which Parity version?**: 
v1.9.2-20180201 beta
 - **Which operating system?**:  
MacOS 
 - **How installed?**: 
 homebrew
 - **Are you fully synchronized?**: 
yes
 - **Which network are you connected to?**: 
poa 
 - **Did you try to restart the node?**: 
no

I run `web3.eth.getBlock('latest')` in the  *Parity/Web3 console*.
It return this error?

```
>
16:14:09.724
web3.eth.getBlock('latest')
✖
16:14:09.730
EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' ".
```

**Comment by marshallstokes** [comment]
I get this error message when attempting to compile contract code. Working on Ropsten network. Does't seem to be a way to compile solidity in this release.

**Comment by hakusaro** [comment]
I'm getting this as well. Chrome version:

```
Google Chrome | 64.0.3282.140 (Official Build) (64-bit)
-- | --
Revision | a06bc1d5e8e285c70078802de990c1719ccc75e8-refs/branch-heads/3282@{#631}
OS | Mac OS X
JavaScript | V8 6.4.388.41
Flash | 28.0.0.161 /Users/hakusaro/Library/Application Support/Google/Chrome/PepperFlash/28.0.0.161/PepperFlashPlayer.plugin
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36
Command Line | /Applications/Google Chrome.app/Contents/MacOS/Google Chrome --flag-switches-begin --site-per-process --enable-features=FramebustingNeedsSameOriginOrUserGesture --disable-features=SharedArrayBuffer --flag-switches-end
Executable Path | /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
Profile Path | /Users/hakusaro/Library/Application Support/Google/Chrome/Default
```

On Parity v1.9.1-20180201 beta. I'm new to this, so I don't know if this is coming from the browser or parity itself, but yeah. More info.

**Comment by andresilva** [comment]
This has been fixed on [v1.9.3](https://github.com/paritytech/parity/releases/tag/v1.9.3) and [v1.8.10](https://github.com/paritytech/parity/releases/tag/v1.8.10).
